build-ps/debian-zenfs/extra/apparmor.d/usr.sbin.mysqld.in

```diff
@@ -28,16 +28,11 @@
   /etc/mysql/** r,
 
 # Allow pid, socket, socket lock file access
-  /var/run/mysqld/mysqld.pid rw,
-  /var/run/mysqld/mysqld.sock rw,
-  /var/run/mysqld/mysqld.sock.lock rw,
-  /var/run/mysqld/mysqlx.sock rw,
-  /var/run/mysqld/mysqlx.sock.lock rw,
-  /run/mysqld/mysqld.pid rw,
-  /run/mysqld/mysqld.sock rw,
-  /run/mysqld/mysqld.sock.lock rw,
-  /run/mysqld/mysqlx.sock rw,
-  /run/mysqld/mysqlx.sock.lock rw,
+  /{,var/}run/mysqld/mysqld.pid rw,
```

Comment:
If we are doing changes to this file, maybe it is worth considering getting this upstream change?
```
# Allow pid, socket, socket lock and other file access
  /run/mysqld/* rw,
  /var/run/mysqld/* rw,
```
commit: 307093c3cd89c91ef0643113738475f7b75ff223

BTW, why do we need the profile duplication? It is harder to maintain 2 versions of the profile for mysqld. I see that extra directories are the same, only maintainer scripts differ. From the maintenance cost point of view, wouldn't it be easier to keep the common stuff only once?